hugo/assets/scss/components/article.scss

```diff
@@ -99,6 +99,11 @@
     &__share {
         flex: 0 0 46px;
         height: 46px;
+
+        &:only-child {
+            margin-left: auto;
+            margin-top: -60px;
```

Comment:
maybe do this in rem since it's (I guess) based on the font-size of 'last-modified'. If a user for instance has a bigger font in the browser (accessibility) using rem's it will have the margin top bases on the font size

Reply:
It's not, but I'll change it to rem anyway